modules/storage-api/src/main/java/org/apache/ignite/internal/storage/AddWriteResultStatus.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.storage;
+
+/** Status of result {@link MvPartitionStorage#addWrite add} of write intent. */
+public enum AddWriteResultStatus {
+    /** Successful add of write intent or replace for same transaction. */
+    SUCCESS,
+
+    /** Failed to add write intent because it already exists for another transaction. */
+    WRITE_INTENT_EXISTS;
```

Comment:
Maybe `TX_ID_MISMATCH` would be easier to read, this name is not specific enough. It means that **conflicting** write intent exists, but that's too long